cache/redisCache.js

```diff
@@ -58,17 +58,22 @@ if (featureToggles.isFeatureEnabled('enableRedisConnectionLogging')) {
   logger.info('Redis Retry Strategy', opts);
 }
 
-const sub = redis.createClient(rconf.instanceUrl.pubsub, opts);
-sub.subscribe(rconf.channelName);
+const subPerspective = redis.createClient(rconf.instanceUrl.pubsubPerspective, opts);
+subPerspective.subscribe(rconf.channelName);
+
+const subBot = redis.createClient(rconf.instanceUrl.pubsubBots, opts);
+subBot.subscribe(rconf.channelName);
```

Comment:
I know I said channel name didn't really matter but upon further reflection I think we should use a separate channel name for the bot pub/sub